src/View/Window.vala

```diff
@@ -220,7 +220,7 @@ namespace Marlin.View {
             });
 
             undo_manager.request_menu_update.connect (undo_redo_menu_update_callback);
-            button_press_event.connect (on_button_press_event);
+            button_press_event.connect_after (on_button_press_event);
```

Comment:
Hm, any reason here, for handling this after the actual event?

Reply:
The existing code handles the "find" function in AbstractDirectoryView after it has handled the other events, so now the "find" code is moved to Windows we connect this way so that the order of processing is unchanged.